src/sql/platform/connection/common/connectionProfileGroup.ts

```diff
@@ -75,8 +75,24 @@ export class ConnectionProfileGroup extends Disposable implements IConnectionPro
 		this._isRenamed = val;
 	}
 
+	public get children(): ConnectionProfileGroup[] {
```

Comment:
As a side note - I also plan on coming back and looking into changing this name. It has two meanings for this class - children in this context are just the child groups but hasChildren/getChildren also includes the connections. So really this property should be called childGroups or something (like I named the private vars).

But as I'm thinking this is a candidate for porting to 1.15 release I wanted to minimize changes here so kept it as-is.